whisper-fine-tuning-event/run_eval_whisper_streaming.py

```diff
@@ -0,0 +1,112 @@
+import argparse
+
+from transformers import pipeline
+from datasets import load_dataset, Audio
+import evaluate
+
+wer_metric = evaluate.load("wer")
+
+
+def is_target_text_in_range(ref):
+    if ref.strip() == "ignore time segment in scoring":
+        return False
+    else:
+        return ref.strip() != ""
+
+
+def get_text(sample):
+    if "text" in sample:
+        return sample["text"]
+    elif "sentence" in sample:
+        return sample["sentence"]
+    elif "normalized_text" in sample:
+        return sample["normalized_text"]
+    elif "transcript" in sample:
+        return sample["transcript"]
+    else:
+        raise ValueError(f"Sample: {sample.keys()} has no transcript.")
```

Comment:
Can we print an error message with the dataset features and expected column names?
```suggestion
        raise ValueError(f"Expected transcript column of either "text", "sentence", "normalized_text" or "transcript". Got sample of " ".join{sample.keys()}. Ensure a text column name is present in the dataset.")
```

Reply:
Done!